vanadium_patches/0071-enable-partitioning-connections-by-default.patch

```diff
@@ -1,47 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Daniel Micay <danielmicay@gmail.com>
-Date: Mon, 8 Mar 2021 16:53:47 -0500
-Subject: [PATCH] enable partitioning connections by default
-
----
- net/base/features.cc | 12 ++++++------
- 1 file changed, 6 insertions(+), 6 deletions(-)
-
-diff --git a/net/base/features.cc b/net/base/features.cc
-index 3f6d23fae985e..512ba5171cf0e 100644
---- a/net/base/features.cc
-+++ b/net/base/features.cc
-@@ -119,27 +119,27 @@ BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
- 
- BASE_FEATURE(kSplitHostCacheByNetworkIsolationKey,
-              "SplitHostCacheByNetworkIsolationKey",
--             base::FEATURE_DISABLED_BY_DEFAULT);
-+             base::FEATURE_ENABLED_BY_DEFAULT);
- 
- BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
-              "PartitionConnectionsByNetworkIsolationKey",
--             base::FEATURE_DISABLED_BY_DEFAULT);
-+             base::FEATURE_ENABLED_BY_DEFAULT);
- 
- BASE_FEATURE(kPartitionHttpServerPropertiesByNetworkIsolationKey,
-              "PartitionHttpServerPropertiesByNetworkIsolationKey",
--             base::FEATURE_DISABLED_BY_DEFAULT);
-+             base::FEATURE_ENABLED_BY_DEFAULT);
- 
- BASE_FEATURE(kPartitionSSLSessionsByNetworkIsolationKey,
-              "PartitionSSLSessionsByNetworkIsolationKey",
--             base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
you mentioned that these features don't exist anymore, but they're still present in the features.cc file. Can you clarify this in the vanadium repo if they're doing something incorrect?